readme.md

```diff
@@ -196,6 +196,27 @@ Default: `10000`
 Timeout in milliseconds for getting a response. Can not be greater than 2147483647.
 If set to `false`, there will be no timeout.
 
+##### onDownloadProgress
+
+Type: `Function`
+
+Download progress event handler. The function takes `progress` and `chunk` arguments.<br>
+The `progress` object contains following elements: `percent`, `transferredBytes` and `totalBytes`. If it's not possible to retrieve the body size, total will be `0`.
+
+For the first call, `chunk` will be always `undefined`.
```

Comment:
Instead of `undefined`, wouldn't it be better to just emit an empty chunk (empty `Uint8Array`)? Then people would not need to implement conditional handling and the types would be simpler.

Reply:
Will do. Great idea.